presto-postgresql/src/test/java/io/prestosql/plugin/postgresql/TestPostgreSqlTypeMapping.java

```diff
@@ -616,6 +667,38 @@ public void testTimestampWithTimeZone(boolean insertWithPresto)
         tests.execute(getQueryRunner(), dataSetup);
     }
 
+    @Test(dataProvider = "testTimestampWithTimeZoneDataProvider")
+    public void testArrayTimestampWithTimeZone(boolean insertWithPresto)
```

Comment:
nit: move this test method below `testArrayTimestamp` (actually below that's method helper method `addArrayTimestampTestIfSupported`)